src/views/UrbitID/ResetKeys/ResetExecute.js

```diff
@@ -98,17 +128,16 @@ export default function ResetExecute({ newWallet, setNewWallet }) {
         walletType === WALLET_TYPES.WALLET_CONNECT ? wcSend : undefined;
 
       try {
-        if (isL2) {
-          console.log(newWallet.value);
+        if (isL2Spawn) {
           // FIXME: the useEffect is called twice, which is fine since the duplicate
           // L2 txs will be discarded
-          //
           await performL2Reticket({
```

Comment:
I think that what we want here is s "performL2SpawnReticket" that does the same as "reticketPointBetweenWallets" with the exception of doing the reset of the spawn-proxy as an L2 transaction, instead of an L1. Otherwise this is going to perform the whole reticketing flow on L2 which is not what we want for solving https://github.com/urbit/bridge/issues/931.

Reply:
Thanks for the feedback @yosoyubik, will make the change